Update to 1.9.0-21

saraedum opened the pull request.

saraedum committed:
Update to 1.9.0-21

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I was trying to look for recipes to lint for you, but it appears we have a merge conflict.
Please try to merge or rebase with the base branch to resolve this conflict.

Please ping the 'conda-forge/core' team (using the @ notation in a comment) if you believe this is a bug.

saraedum committed:
Merge remote-tracking branch 'origin/main' into 19021

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe```) and found it was in an excellent condition.

saraedum commented:
@conda-forge-admin, please rerender.

Commit:
MNT: Re-rendered with conda-build 3.22.0, conda-smithy 3.21.2, and conda-forge-pinning 2022.09.28.16.00.31

saraedum committed:
Fix SHA

saraedum committed:
Rebase patch from SageMath

saraedum commented:
Build currently fails with

```
icas.cc: In function 'int main(int, char**)':
icas.cc:1435:9: error: 'localisation' is not a member of 'xcas'
 1435 |   xcas::localisation(); // change by L. Marohnić
      |         ^~~~~~~~~~~~
```

saraedum commented:
There's a post in the forums with~out~ a solution at https://xcas.univ-grenoble-alpes.fr/forum/viewtopic.php?t=2789

saraedum committed:
Fix compilation

see https://xcas.univ-grenoble-alpes.fr/forum/viewtopic.php?t=2789

saraedum commented:
@isuruf is there a strong reason why we `--disable-gui` here?

I guess we don't want to require GL and such to be available?

saraedum committed:
Disable GUI

building with --enable-gui requires GL and such which we don't want to
reuire (yet.)

saraedum committed:
Need gnu99 flags to get clock_gettime() on macOS

saraedum committed:
Do not use clock_gettime() which is not available on old verions of macOS

saraedum committed:
Fix compilation on macOS

saraedum commented:
Is this something in conda or giac? :boom: 

```
/Users/runner/miniforge3/etc/profile.d/conda.sh: line 9:  3442 Segmentation fault: 11  "$CONDA_EXE" $_CE_M $_CE_CONDA "$@"
/Users/runner/miniforge3/lib/python3.10/multiprocessing/resource_tracker.py:224: UserWarning: resource_tracker: There appear to be 1 leaked semaphore objects to clean up at shutdown
  warnings.warn('resource_tracker: There appear to be %d '
```

saraedum commented:
Rerunning the job made it go through.
